.github/workflows/test-os.yml

```diff
@@ -80,15 +79,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2022, windows-2025]
+        os: [windows-2022]
         worker:
           - containerd
         pkg:
           - ./client
           - ./cmd/buildctl
           - ./worker/containerd
-          - ./solver
-          - ./frontend
```

Comment:
I think we should keep them in case new tests for windows in these packages are added in the future.

Reply:
Sure, reverted.